frontend/src/pages/BYONImages/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { BYONImage } from '~/types';
+
+export const convertBYONImageToK8sResource = (image: BYONImage): K8sResourceCommon => ({
+  kind: 'ImageStream',
+  apiVersion: 'image.openshift.io/v1',
+  metadata: {
+    name: image.name,
+    annotations: {
+      'openshift.io/display-name': image.display_name,
+    },
+  },
+});
+
+// 1 = Enabled, 0 = Disabled, -1 = Error
+export const getEnabledStatus = (image: BYONImage): number =>
+  image.visible && !image.error ? 1 : image.error ? -1 : 0;
```

Comment:
Please use an enum -- magic numbers, especially ones you want to type a comment about, should be handled with descriptive code (like enums)